test/common/router/retry_state_impl_test.cc

```diff
@@ -1518,6 +1518,11 @@ TEST_F(RouterRetryStateImplTest, ParseRetryGrpcOn) {
   EXPECT_EQ(result.first, 224);
   EXPECT_TRUE(result.second);
 
+  config = "cancelled,deadline-exceeded,resource-exhausted,unknown";
+  result = RetryStateImpl::parseRetryGrpcOn(config);
+  EXPECT_EQ(result.first, 32992);
```

Comment:
This would be easier to read as a hexadecimal constant (even though the others weren't written this way)